programs/compressed-token/src/process_mint.rs

```diff
@@ -1,3 +1,5 @@
+#[cfg(target_os = "solana")]
```

Comment:
why is this necessary?

Reply:
cargo clippy throws unused import otherwise because `get_cpi_signer_seeds` is only used in target_os = "solana"